docs/en/user-guide/layerexplanations/twwproject.rst

```diff
@@ -0,0 +1,106 @@
+Important to know on the TWW-project
+====================================
+
+Project Properties
+------------------
+There are several important properties in a TWW-project:
+
+.. figure:: images/project_properties.jpg
+
+Data Sources
+^^^^^^^^^^^^
+Important Settings:
+
+* Transaction mode has to be **Automatic Transaction Gropus**
+
+* **Evaluate default values on provider side** has to be checked
+
+.. figure:: images/properties_datasources.jpg
+
+Relations
+^^^^^^^^^^^^
+Relations are necessary for the Feature Attributes Windows to referende to connected tables (e.g. you can edit connected maintenance_events in Feature_Attribut window in vw_tww_reach).
+Where possible, in TWW are value relations used and not project relations. Value Relations ar used for Value Lists, but also for organisations and for pipe_profiles.
+
+**Value Relations** work perfect also with Multiedit, but is slow, if there are a lot records in the related table or list. And you can not choose a record on the map to connect to it (e.g to define a manhole-catchment area-connection).
+
+**Project relations** allow to select a record on the map and is faster with big datasets. But there is a serious bug when using Multiedit (as of today with QGIS 3.34.4).
+If there is a good advantage to work with Multiedit, it's a good idea to define a style, that has no relation-field in the Feature Attributs window.
+
+
+Variables
```

Comment:
Moved to an attention point that can be removed when QGIS behaviour becomes stable again when dealing with Project relations